test/navigation/NavigatorSpec.scala

```diff
@@ -660,6 +686,60 @@ class NavigatorSpec extends SpecBase with BeforeAndAfterEach {
 
             }
 
+            "NIPHL is not authorised and has NIPHL assesments" in {
+
+              val categoryInfoWithNiphlAssessments = CategorisationInfo(
+                "1234567890",
+                Seq(category1Niphl),
```

Comment:
Missing the empty cat 2 needed for a NIPHLs assessment

Reply:
Updated